### ISSUE TITLE
MathJax.Formula not updating on prop change

### ISSUE THREAD

**Issue by eriknikulski** [asnunes/mathjax3-react#2]
Hi,
thanks a lot for providing this package.

But I have the problem when updating my variable `input.value` the component `MathJax.Formula` seems to not be updated. Instead I just get the normal string with the dollar signs in my html.

```jsx
<MathJax.Provider>
    <MathJax.Formula formula={'$$' + input.value + '$$'} />
</MathJax.Provider>
```

**Comment by asnunes** [comment]
Hi, Erik. Thank you for your report!

This bug was fixed on version v1.0.7 now available on NPM.
Basic input example implementation is also available on [example directory](https://github.com/asnunes/mathjax3-react/blob/master/src/examples/formula/CustomInput.js) and [live demo](https://asnunes.github.io/mathjax3-react/).

**Comment by eriknikulski** [comment]
Thanks for fixing it!

I added

```js
options={{
  loader: {
    load: ['[tex]/noerrors']
  },
  tex: {
    packages: {
      '[+]': ['noerrors']
    },
  },
}}
```
 to my MathJax.Provider and now I'm getting this error:

```
TypeError: MathJax.typesetPromise is not a function 2 useMathJax.js:10
    updateMathContent useMathJax.js:10
    useMathJax useMathJax.js:6
    React 6
    unstable_runWithPriority scheduler.development.js:653
    React 3
    workLoop scheduler.development.js:597
    flushWork scheduler.development.js:552
    performWorkUntilDeadline scheduler.development.js:164

```
